### PR TITLE
Update readme and contributor docs

### LINKED ISSUE #61: rewrite README
Rewrite the repo's README to describe current practice.

### PR DESCRIPTION
**Pull request by meker12**
Fixes #61.
- Update readme to reflect current content
- Add contributing.md with project description
  and instructions for updating SDK doc and doc menu
- Add githubing instructions
- Rename SDK doc conventions to quickstart-doc-conventions
- Link SDK doc conventions from SDK update instructions in
  contributing.md

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-cafc7fecd4/docs/).

**Comment by meker12** [comment]
@kmsholcomb  Please review the updated files when you have time.  Thank you!

**Comment by kmsholcomb** [comment]
Will do, @meker12

**Comment by kmsholcomb** [comment]
Hi @meker12. I know that you prefer comments in the files, or PRs, but in the interest of time, I am posting a PDF that contains all the markup I made in four Word files. Please let me know if you have any questions. Thanks!

[quickstart_docs_editsKH.pdf](https://github.com/rackerlabs/docs-quickstart/files/384775/quickstart_docs_editsKH.pdf)

**Comment by catlook** [comment]
@meker12 when you're finished applying any changes, I'll merge!

**Comment by kimtryce** [comment]
@zack-shoylev when you have a good stoping point do you mind giving a quick tech/process review of this doc?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+1. Verify that the HTML syntax is valid.
+2. Commit and push changes to your fork.
+
+## Submitting changes
+
+When you've completed your changes, submit a pull request so that someone on the Developer Experience team can review them.
+
+- Minor updates and corrections get a quick review to ensure that content is
+  error-free and doesn't introduce other issues.
+- More complex changes or additions require both technical and editorial
```
Comment (zack-shoylev) [inline CONTRIBUTING.md]: Do PR submitters have to do anything extra here?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-5e2889201d/docs/). :bow:

**Comment by meker12** [comment]
Applied edits and clarified that the Developer Experience team will merge the commit per PR review.  @zack-shoylev When you have a chance can you review the most recent version.  Thanks!

**Comment by zack-shoylev** [comment]
:+1:

**Pull request merged**